plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ansible_galaxy_install
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Install Ansible roles or collections using ansible-galaxy
+version_added: 3.5.0
+description:
+  - This module allows the installation of Ansible collections or roles using C(ansible-galaxy).
+notes:
+  - >
+    B(Ansible 2.9/2.10): The C(ansible-galaxy) command changed significantly between Ansible 2.9 and
+    ansible-base 2.10 (later ansible-core 2.11). See comments in the parameters.
+  - >
+    B(Python 2.6): The upcoming version 2.12 of C(ansible-core) is not compatible with Python 2.6, specifically the
+    C(ansible-galaxy) command will not compile.
+options:
+  type:
+    description:
+    - The type of installation performed by C(ansible-galaxy).
+    - If I(type) is C(both), then I(requirements_file) must be passed and it may contain both roles and collections.
+    - "Note however that the opposite is not true: if using a I(requirements_file), then I(type) can be any of the three choices."
+    - "B(Ansible 2.9): The option C(both) will have the same effect as C(role)."
```

Comment:
How about reading `ansible-galaxy --version` at the beginning of the module and failing if `type=both` is specified? That way users will be prevented from having surprises.

Reply:
done